problem_set_1.py

```diff
@@ -1,39 +1,62 @@
 # 1. Set the variable `given_name` to the string "Addison".
+given_name = 'Addison'
 
 
 # ------------------------------------------------------------------------------
 # 2. You have 20 candies that you must divide equally among 6 people. How many candies will be left over?
 # Set variables for `candies`, `people`, `left_over` to make your tests pass.
+candies = 20
+people = 6
+left_over = 20 % 6
 
 
 # ------------------------------------------------------------------------------
 # 3. Create a function called `greeting` that returns "Hello, <name>!",
 # where <name> is the name given as an argument to the function.
-
+def greeting():
+    print('Hello <name>!')
```

Comment:
The angle brackets here `<>` indicate that the whole thing will be replaced. A variable will go there instead, and the value of that variable will change, depending on what arguments (actual values) are passed when the function is called.